mock-core-configs/etc/mock/templates/circlelinux-8.tpl

```diff
@@ -0,0 +1,150 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'el8'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '8'
+config_opts['package_manager'] = 'dnf'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['bootstrap_image'] = 'docker.io/circlelinuxos/circlelinux:8'
+
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+metadata_expire=0
+mdpolicy=group:primary
+best=1
+install_weak_deps=0
+protected_packages=
+module_platform_id=platform:el8
+user_agent={{ user_agent }}
+
+
+[baseos]
+name=Circle Linux $releasever - BaseOS
+mirrorlist=https://mirrorlist.cclinux.org/mirrorlist?arch=$basearch&repo=BaseOS
+#baseurl=https://mirror.cclinux.org/$contentdir/$releasever/BaseOS/$basearch/os/
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-circleofficial
```

Comment:
Please add this key to https://github.com/xsuchy/distribution-gpg-keys/ and then use the path from this package.

Reply:
OK , I added by the pr https://github.com/xsuchy/distribution-gpg-keys/pull/65

Reply:
Now, please bump up the requires in mock-core-configs.spec to require distribution-gpg-keys >= 1.72 to make sure the keys are there.

Reply:
OK, get it.
Thank you

Reply:
Just search for the `Requires: distribution-gpg-keys ...` statement, and increase the number, please.

Reply:
OK , mock-core-configs.spec  had increased, thank you!